scalding-quotation/src/test/scala/com/twitter/scalding/quotation/package.scala

```diff
@@ -0,0 +1,14 @@
+package com.twitter.scalding
+
+import org.scalatest.MustMatchers
+import org.scalatest.FreeSpec
+
+package object quotation {
+  def cls[T](implicit ct: reflect.ClassTag[T]) = ct.runtimeClass.getName
+  trait Test extends FreeSpec with MustMatchers
+
+  val person = TypeReference(cls[Person])
+  val name = person.andThen("name", cls[String])
```

Comment:
ahh, here it is. Can we not do this and instead add an explicit import? It makes it harder to read the tests without a clear benefit if you ask me.

Reply:
I've moved the projections to the `Person` companion object